src/LegacyRecord.php

```diff
@@ -66,82 +43,44 @@ class LegacyRecord implements ModelInterface, HasSchemaInterface
      * models using this parameter. Every time this
      * model is queried, it will load its referenced
      * models together.
+     *
+     * @var array
```

Comment:
```suggestion
```
I don't think this is necessary.
The type it's already explicit on property declaration.